web/packages/teleterm/src/ui/services/workspacesService/documentsService/documentsService.ts

```diff
@@ -479,6 +493,25 @@ export class DocumentsService {
     return this.getState().documents.filter(i => i.uri !== uri);
   }
 
+  /**
+   * Finds an existing doc using findExisting and opens it. If there's no existing doc, uses
+   * createNew to add a new doc and then opens it.
+   */
+  openExistingOrAddNew(
```

Comment:
Finally decided to create a method for that after years of writing this by hand. 😂